edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.CloudProxy/ServiceProxy.cs

```diff
@@ -21,6 +21,8 @@ public ServiceProxy(IDeviceScopeApiClient securityScopesApiClient)
 
         public IServiceIdentitiesIterator GetServiceIdentitiesIterator() => new ServiceIdentitiesIterator(this.securityScopesApiClient);
```

Comment:
Do we need to keep this part and all the code attached to it? Seems that nested is a super set of this one.

Reply:
It seems this method is no longer used in DeviceScopeIdentitiesCache line158, which is replaced by GetNestedServiceIdentitiesIterator method.